test/revert.spec.ts

```diff
@@ -3146,9 +3310,7 @@ describe(`Reverts (Seaport v${VERSION})`, function () {
         marketplaceContract
           .connect(owner)
           .matchAdvancedOrders([order, order2], [], fulfillments)
-      ).to.be.revertedWith(
-        "panic code 0x11 (Arithmetic operation underflowed or overflowed outside of an unchecked block)"
-      );
+      ).to.be.revertedWithPanic(PANIC_CODES.ARITHMETIC_UNDER_OR_OVERFLOW);
```

Comment:
This is another new method that is available in Hardhat Chai Matchers and not in waffle.